### ISSUE TITLE
Java 11 now required to use Java 8 functions

### ISSUE THREAD

**Issue by anthonychu** [microsoft/vscode-azurefunctions#2300]
The VS Code Java extension now requires Java 11 to run. For Java 8 apps, both JDK 8 and 11 must be on the machine to run the app and to get proper language services like intellisense.

Is there something this extension can do to help detect missing JDK 11 and guide the customer to the right setup?

Related: https://github.com/microsoft/vscode-dev-containers/issues/459

/cc @apawast

**Comment by EricJizbaMSFT** [comment]
Are you sure Java 8 is supported at all in VS Code? The error in the issue you referenced is coming from the Java extension. If they don't support 8, there's nothing we can do about that. Seems like users might have to switch their projects to 11 once we support it https://github.com/microsoft/vscode-azurefunctions/issues/2033

**Comment by anthonychu** [comment]
Yep it’s still supported. The problem is the Java extension now requires JDK 11 to run, while Java 8 Functions requires JDK 8 to be installed also. Java Home then needs to be configured properly in VS Code [like this](https://github.com/redhat-developer/vscode-java/issues/1543#issuecomment-663382692) for the Java language service in VS Code to work for Java 8 function apps. Wondering if there’s anything we can do to help the customer configure the right settings in the workspace or maybe just point them in the right direction to fix it themselves.
